ar.json

```diff
@@ -1,5 +1,9 @@
 {
   "ar": {
+    "after": "بعد",
+    "before": "قبل",
+    "is": "يكون",
+    "isNot": "ليس",
```

Comment:
These are wrong PrimeNG should be using the following which are already defined in the file...
```
    "dateAfter": "التاريخ بعد",
    "dateBefore": "التاريخ قبل",
    "dateFormat": "شهر/يوم/سنة",
    "dateIs": "التاريخ هو",
    "dateIsNot": "التاريخ ليس",
```

See PrimeNG bug: https://github.com/primefaces/primeng/issues/14446